src/main/java/org/springframework/cloud/deployer/spi/kubernetes/DeploymentPropertiesResolver.java

```diff
@@ -480,11 +481,15 @@ Container getInitContainer(Map<String, String> kubernetesDeployerProperties) {
 			String commands = PropertyParserUtils.getDeploymentPropertyValue(kubernetesDeployerProperties,
 					this.propertyPrefix + ".initContainer.commands");
 
+			List<VolumeMount> vms = this.getVolumeMounts(PropertyParserUtils.getDeploymentPropertyValue(kubernetesDeployerProperties,
```

Comment:
given the spliting of `getVolumeMounts` below, i think this would be more clear/consistent if there was a `getInitContainerVolumeMounts(Map<String, String>)` as with `getVolumeMounts(Map<String, String>)`